fern/docs/pages/integrations/webhooks-plan-changed.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Plan Changed Webhook
+
+slug: integrations/webhooks
+---
+
+The `company.plan_change` webhook is one of the most important triggers for building sales and customer success workflows. It fires whenever a company's plan changes, whether an upgrade, downgrade, or the end of a trial period.
+
+Some common use cases for `company.plan_changed` webhook include:
+- **Identifying PLG upgrades:** Automatically notify your team or trigger automations when a self-serve customer upgrades their plan, so you can reach out to nurture and support the account.
+- **Tracking trial conversions and endings:** Get alerted when a company's trial ends, enabling you to follow up or offer interventions to encourage conversion.
+- **Catching downgrades:** Proactively inform Customer Success (CS) when an account downgrades, giving your team a chance to understand any pain points and try to save the account.
+
+By listening for the `plan_changed` webhook, you can automate account lifecycle tasks and keep key teams informed about important moments in your customer journey.
+
+## Setup
+
+The `company.plan_change` webhook is set up like all other webhooks in Schematic. Please follow the [general webhook setup instructions](/integrations/webhooks) to create an endpoint and select the `company.plan_changed` trigger.
+
+## Fields
+
+All Schematic webhooks are strongly typed in each of our [SDKs](/developer_resources/sdks/overview). 
```

Comment:
cc @cbrady this is something I've been thinking about recently. kind of irrelevant to this PR but thsi maade me think of it

I wonder if we should do more in our SDKs to show the types for each webhook. e.g. a minimal way would be to have a table in each readme with each row being a webhook type and a link to the type one would expect for the body of this webhook. A more robust helper would be an actual tool that helps with deserializing to the appropriate type, but this is pretty complex since you don't know what type to return until you read the webhook.